lib/module.js

```diff
@@ -112,6 +112,9 @@ function readPackage(requestPath) {
     return false;
   }
 
+  if (json === '')
```

Comment:
Does V8 make this and `json.length === 0` equivalent instruction-wise?

Reply:
`s.length === 0` is probably a bit slower.  The empty string is a singleton so `s === ''` is basically a pointer comparison, whereas `s.length` needs to validate `s` first (although I expect it will be a fast path protected by a guard once optimized.)